package.json

```diff
@@ -36,25 +36,23 @@
     "Wrapper",
     "Charts"
   ],
-  "main": "dist/vue-chartjs.cjs",
-  "module": "dist/vue-chartjs.mjs",
-  "jsnext:main": "es/index.js",
-  "typings": "types/index.d.ts",
-  "files": [
-    "src",
-    "dist",
-    "es",
-    "types/*.d.ts"
-  ],
+  "main": "./src/index.js",
+  "typings": "./dist/index.d.ts",
+  "publishConfig": {
+    "main": "./dist/index.cjs",
+    "module": "./dist/index.js",
+    "directory": "package"
+  },
   "scripts": {
+    "prepublishOnly": "pnpm lint && pnpm test && pnpm build && rm -rf ./package && clean-publish",
```

Comment:
1) `pnpm test` already with `pnpm lint` 
2) please use del-cli to remove directory. `rm` doesn't work on windows

Reply:
fixed prepublishOnly script and added del-cli